src/Docfx.App/PdfBuilder.cs

```diff
@@ -270,7 +270,12 @@ static string ExpandTemplate(string? pdfTemplate, int pageNumber, int totalPages
         if (numberOfPages is 0)
             return;
 
-        var producer = $"docfx ({typeof(PdfBuilder).Assembly.GetCustomAttribute<AssemblyVersionAttribute>()?.Version})";
+        var entryAssembly = Assembly.GetEntryAssembly() ?? typeof(PdfBuilder).Assembly;
```

Comment:
Do we want to use entry assembly if it is called using `Docfx.App` package?

Reply:
> Do we want to use entry assembly if it is called using Docfx.App package?
There is no particular reason to stick to it.
It just curious about showing `Docfx.App` assembly file version as `docfx` version.

I'll fix new commit about this.